python/ngen_conf/tests/test_conf.py

```diff
@@ -0,0 +1,46 @@
+import pytest, json
+from pathlib import Path
+from ngen.config.realization import NgenRealization
+from ngen.config.hydrofabric import NGenNexusFile, NGenCatchmentFile
+
+@pytest.fixture()
+def catchmentdata():
```

Comment:
As an aside, just a cool little nugget about `pytest`, fixtures can take other fixtures as arguments. So you could have a `test_dir` fixture that the `catchmentdata`, `nexusdata`, and `realizationdata` fixtures each use used. You pass them in just like you would a fixture to a test. Its totally fine how it is, just trying to teach a little along the way!

Reply:
Took your suggestion and made this change